nuxt/layouts/ddt.vue

```diff
@@ -102,16 +103,18 @@ export default {
     }
   },
   computed: {
-    isAllowed () {
-      // console.log('this.$user?.profile: ', this.$user.profile)
-      return (this.$user?.profile?.side === 'etat' && this.$user?.profile?.verified) || this.$user.profile.is_admin || this.$isDev
+    isVerified () {
+      return this.$user?.profile?.verified || this.$user.profile.is_admin
```

Comment:
C'est l'occasion d'enlever la spécificité `is_admin` qui ne me semble pas utile.

```suggestion
      return this.$user?.profile?.verified
```